src/pages/Login.tsx

```diff
@@ -22,6 +25,7 @@ function Login() {
       .then(response => {
         setRedirect(HOME_PATH);
         setCookies(ACCESS_TOKEN_COOKIE_NAME, response?.data?.token);
+        setCookies(USER_ID_COOKIE_NAME, response?.data?.user?.id);
```

Comment:
*  Do the same thing for logout, setting to null
* Add this to the isUserLoggedIn method too